workspace-hack/Cargo.toml

```diff
@@ -77,15 +74,13 @@ phf_shared = { version = "0.11" }
 rand = { version = "0.8", features = ["min_const_gen", "small_rng"] }
 rand_chacha = { version = "0.3" }
 rand_core = { version = "0.6", default-features = false, features = ["std"] }
-schemars = { version = "0.8", features = ["bytes", "chrono", "uuid1"] }
+schemars = { version = "0.8", features = ["bytes", "chrono", "uuid", "uuid1"] }
```

Comment:
Is there an easy way to use `cargo hakari` to figure out where the `uuid` feature is being added?  We only enable `uuid1` in Crucible's `Cargo.toml`.

Reply:
I think you can use `cargo tree -e features` to see which features are enabled on each dependency.

Reply:
Oh I fixed this in maghemite: https://github.com/oxidecomputer/maghemite/pull/191